app_dart/bin/validate_task_ownership.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:cocoon_service/src/request_handlers/flaky_handler_utils.dart';
```

Comment:
Should we move some the getTestowner code out to a more generic util file?

Reply:
I would prefer to do that in a separate PR.